Source/WebCore/html/HTMLLabelElement.cpp

```diff
@@ -103,6 +117,12 @@ HTMLFormElement* HTMLLabelElement::form() const
 
 HTMLFormElement* HTMLLabelElement::formForBindings() const
 {
+    if (document().settings().shadowRootReferenceTargetEnabled()) {
+        auto* form = this->form();
```

Comment:
Why not RefPtr?

Reply:
It should have been, but this code is gone now anyway.